soccer/gameplay/tactics/our_placement.py

```diff
@@ -59,11 +59,20 @@ def execute_dribble(self):
 
     def on_exit_dribble(self):
         self.robot.set_dribble_speed(0)
-        self.remove_subbehavior('dribble')
+        self.remove_all_subbehaviors()
+        print("EXIT DRIBBLE")
 
     def on_enter_pause(self):
+        self.robot.is_ball_placer = True
         self.pause_time = time.time()
 
+        #without this the robot tries to leave the circle in a random direction
+        self.robot.set_max_speed(0.01)
+
+
+    def on_exit_pause(self):
+        self.robot.set_max_speed(5)
```

Comment:
I'm not even sure if they will be necessary any more, they were mostly to deal with our issues with motion control last year.